src/vm/excep.cpp

```diff
@@ -5138,7 +5138,12 @@ LONG InternalUnhandledExceptionFilter(
 
 } // LONG InternalUnhandledExceptionFilter()
 
+
+#ifdef PLATFORM_WINDOWS
+static bool s_useEntryPointFilter = CLRConfig::GetConfigValue(CLRConfig::INTERNAL_UseEntryPointFilter) != 0;
```

Comment:
We try to avoid running any interesting code in static constructors. This should be part of `ParseUseEntryPointFilter` or a similar method that is explicitly called at some point.

Reply:
We find the reason for the `GetUseEntryPointFilter()` which was removed...

Reply:
The weird part of `GetUseEntryPointFilter` was that it returned `bool&`. Having `GetUseEntryPointFilter` that returns `bool` is fine.

Reply:
I have restructured this.  To avoid returning a `bool&` or interesting code in global statics.